openapi/common.go

```diff
@@ -307,9 +322,9 @@ func convertObjectToLocalStateData(property *SpecSchemaDefinitionProperty, prope
 		}
 	}
 
-	if len(objectInput) == 0 {
```

Comment:
restore original behaviour when input is empty, output is expected to be empty map instead of nil

Reply:
The original commit comment was "fix: Missed regression caught by tests - local state no longer updated with default values on property omission during create" - best to check for regression of that behaviour